knimin/lib/tests/test_parse.py

```diff
@@ -68,6 +68,95 @@ def test_parse_plate_reader_output(self):
               0.719, 0.622, 4.342, 4.166, 0.633, 11.101]])
         npt.assert_almost_equal(obs, exp)
 
+    def test_parse_qpcr_object(self):
+        obs = parse_qpcr_object(QPCR_OBJECT)
+        exp = [
+            [10.73, 7.3, 6.77, 6.66, 7.44, 12.1, 6.63, 6.53, 9.32, 6.34, 6.05,
+             7.77, 11.47, 15.52, 12.6, 11.18, 21.37, 15.07, 11.45, 18.32,
+             12.68, 11.52, 11.34, 24.32],
+            [6.61, 6.27, 12.11, 7.15, 6.21, 7.31, 6.87, 5.98, 6.43, 6.59, 6.5,
+             8.07, 7.23, 7.82, 10.09, 15.68, np.nan, np.nan, 14.78, 11.54,
+             np.nan, np.nan, np.nan, np.nan]]
+        npt.assert_almost_equal(obs, exp)
+
+        # errrors
+
+        with self.assertRaises(ValueError) as ctx:
```

Comment:
Possible for assertRaisesRegexp since there are multiple ways Value error can be thrown?